helix-loader/src/grammar.rs

```diff
@@ -433,11 +433,12 @@ fn build_tree_sitter_library(
     for (key, value) in compiler.env() {
         command.env(key, value);
     }
+
     command.args(compiler.args());
     // used to delay dropping the temporary object file until after the compilation is complete
     let _path_guard;
 
-    if cfg!(all(windows, target_env = "msvc")) {
+    if compiler.is_like_msvc() {
```

Comment:
this is unrelated but allows us to cross-compile helix (otherwise it uses the environment of the build script here)